NETWORK/NetworkGetPresenceInviteInviter.md

```diff
@@ -6,11 +6,13 @@ aliases: ["0x4962CC4AA2F345B7"]
 
 ```c
 // 0x4962CC4AA2F345B7 0x19EC65D9
-Any NETWORK_GET_PRESENCE_INVITE_INVITER(Any p0);
+char* NETWORK_GET_PRESENCE_INVITE_INVITER(int inviteIndex);
+```
+```
```

Comment:
Please don't make descriptions in code blocks as outlined [here](https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md).